integration-guides/geotab.mdx

```diff
@@ -9,6 +9,10 @@ To connect your Geotab telematics data with PathStack, you will need the followi
 
 These credentials are the same ones used to log in to the MyGeotab web portal. We recommend creating a new user with read-only access for the PathStack connection. To do this, follow these step:
 
+<Note>If your account has operational hours or geospatial restrictions set on the account then we will be only able to collect data when these conditions are met</Note>
+
+<Note>Geotab includes any idling that takes place after the end of a trip and before the start of the next trip to be part of the current trip. We capture this idling data as a separate trip in our system.</Note>
```

Comment:
> Geotab **includes** any idling that takes place after the end of a trip and before the start of the next trip **to be** part of the current trip.

"Includes ... to be" isn't correct.  "Includes ... as" does, or "Includes ... in".

Geotab **includes** any idling that takes place after the end of a trip and before the start of the next trip **in** the current trip.